DOC-2270_TINY-10581: The toolbar width was miscalculated for the inline editor positioned inside a scrollable container. (#3097)
tinymce/tinymce-docs: modules/ROOT/pages/7.0-release-notes.adoc

// CCFR here.

=== The toolbar width was miscalculated for the inline editor positioned inside a scrollable container.
// #TINY-10581

Previously in {productname}, the inline editor's toolbar width was calculated using the `outerContainer` width.

> Consequently, the toolbar was given a width of `0`, resulting in the toolbar collapsing to the minimum width.

Reviewer: here actually the problem was that only in some cases the calculation of the `outerContainer` width results in a `0`

Reviewer 2: ```suggestion
Previously, the inline editor's toolbar width was calculated using the `outerContainer` width.

As a consequence, in some cases, the calculation of the `outerContainer` width resulted in a width of `0`, causing the toolbar to collapse to the minimum width.
```

Reviewer 2: @lorenzo-pomili Please re-review the suggested change and approve it your happy with this so i can merge.
Thanks,
